freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -42,7 +43,7 @@
 
     @Override
     public SyncOperationStatus synchronizeUser(SynchronizeUserRequest request) {
-        String userCrn = checkUserCrn();
+        String userCrn = FMSUtil.checkUserCrn(threadBaseUserCrnProvider.getUserCrn());
```

Comment:
doesn't compile, most probably because it's missing FMSUtil class
but anyway, this method should be in CrnService as it seems like it belongs there